.cspell-partial.json

```diff
@@ -1,5 +1,5 @@
 {
   "ignorePaths": ["**/perception/**", "**/planning/**", "**/sensing/**"],
-  "ignoreRegExpList": [],
+  "ignoreRegExpList": ["perception/bytetrack/lib/**"],
```

Comment:
May I ask why here instead of "ignorePaths"?

Reply:
Oh, it's my mistake!
Thank you

Reply:
FIxed in https://github.com/autowarefoundation/autoware.universe/pull/4517/commits/86c558060b0e9e4a49f4f99f298dd1753d5180c1